### ISSUE TITLE
`EOFError: read end of file` when adding larger number of workers

### ISSUE THREAD

**Issue by jtackm** [JuliaGraphs/LightGraphs.jl#937]
I regularly (but not always) get the following error when adding 20 or more parallel workers before importing LightGraphs (testing with 5 or 10 workers yields no error):

```julia
julia> VERSION
v"0.6.2"

julia> Pkg.status("LightGraphs")
 - LightGraphs                   0.12.0

julia> addprocs(20)
julia> using LightGraphs

signal (11): Segmentation fault
while loading no file, in expression starting on line 0
unknown function (ip: 0x7f7d0000d610)
unknown function (ip: 0x7f7e0000dc02)
jl_call_fptr_internal at /buildworker/worker/package_linux64/build/src/julia_internal.h:339 [inlined]
jl_call_method_internal at /buildworker/worker/package_linux64/build/src/julia_internal.h:358 [inlined]
jl_apply_generic at /buildworker/worker/package_linux64/build/src/gf.c:1926
jl_apply at /buildworker/worker/package_linux64/build/src/julia.h:1424 [inlined]
jl_f__apply at /buildworker/worker/package_linux64/build/src/builtins.c:426
jl_f__apply_latest at /buildworker/worker/package_linux64/build/src/builtins.c:464
message_handler_loop at ./distributed/process_messages.jl:161
process_tcp_streams at ./distributed/process_messages.jl:118
#99 at ./event.jl:73
unknown function (ip: 0x7f7e0000058f)
jl_call_fptr_internal at /buildworker/worker/package_linux64/build/src/julia_internal.h:339 [inlined]
jl_call_method_internal at /buildworker/worker/package_linux64/build/src/julia_internal.h:358 [inlined]
jl_apply_generic at /buildworker/worker/package_linux64/build/src/gf.c:1926
jl_apply at /buildworker/worker/package_linux64/build/src/julia.h:1424 [inlined]
start_task at /buildworker/worker/package_linux64/build/src/task.c:267
unknown function (ip: 0xffffffffffffffff)
Allocations: 1670944 (Pool: 1669689; Big: 1255); GC: 1
Worker 19 terminated.
ERROR (unhandled task failure): EOFError: read end of file
```

Any idea, what could be causing this?

**Comment by sbromberger** [comment]
Hard to say - not sure this is a LightGraphs problem. Does it happen with any other packages of similar size?

Also, you're probably going to want `@everywhere using LightGraphs` if you want to be able to call LG functions on other processors.

```
bromberger1@spc01:~$ ~/dev/julia6-bin/bin/julia
               _
   _       _ _(_)_     |  A fresh approach to technical computing
  (_)     | (_) (_)    |  Documentation: https://docs.julialang.org
   _ _   _| |_  __ _   |  Type "?help" for help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 0.6.2 (2017-12-13 18:08 UTC)
 _/ |\__'_|_|_|\__'_|  |  Official http://julialang.org/ release
|__/                   |  x86_64-pc-linux-gnu

julia> addprocs(20);

julia> @everywhere using LightGraphs

julia> @everywhere g = WheelGraph(10)

julia>
```
I'd recommend filing an issue over in the main Julia repo.

**Comment by jtackm** [comment]
Thanks a lot for your quick feedback. Using ```@everywhere``` indeed runs without error. However, as usual the example I gave was a bit simplified: I'm actually importing a package that itself depends on LightGraphs (via ```using```), and if I import that one I still get the same error (albeit less frequently when doing ```@everywhere using Package``` compared to a simple ```using Package```).

Following your advice I tested ```DifferentialEquations```. Same thing there, so I think opening and issue at the main repo makes sense. Thanks again for your time!
